### DISCUSSION TITLE
More 486 cpus

### DISCUSSION THREAD

**Discussion by ghost** [86Box/86Box#4546] [Ideas]
486 dlc
486 sl
486 gx
dx2wb and dx4wb
-s variants

**Comment by jriwanek** [comment]
The 486SLC, DLC and -S variants are already implemented and have been for quite some time.

The GX is an embedded variant of the SX, we don't have any machines which use it, could you suggest one?

**Reply by Torinde** [reply]
I'm interested in:
- Cyrix Cx486DX4 (M9) - up to 120MHz ([ST486DX4V12HS](https://theretroweb.com/cpus/2035)) - released after Cyrix 5x85 (which has CPUID, has to be enabled via CCR4), does it have CPUID?
- [IBM 486BL/2/3](https://en.wikipedia.org/wiki/IBM_386SLC#IBM_486BL_(Blue_Lightning)), using i486 and released after Pentium, so maybe have CPUID?
- RDC [R86xx](http://www.sima.com.tw/download/R8610_D06_20051003.pdf)/Vortex86SX - 486SX instruction set
- 386EX/CX - embedded variants with 26-bit addresses
- [overclocked 5x86](https://github.com/86Box/86Box/discussions/5454#discussioncomment-14571594)

**Comment by ghost** [comment]
And WB variants

**Reply by ghost** [reply]
The WB variant would make basically no difference as we do not emulate cache memory.
